Add support for WPML (plugin / add-ons) (#42)
junaidbhura/composer-wp-pro-plugins: README.md

      "dist": {
        "type": "zip",
        "url": "https://wpml.org/"
      },
      "require": {
        "junaidbhura/composer-wp-pro-plugins": "*"
      }
    }
  },
  {
    "type": "package",
    "package": {
      "name": "junaidbhura/wpml-sticky-links",
      "version": "<version_number>",
      "type": "wordpress-plugin",
      "dist": {
        "type": "zip",
        "url": "https://wpml.org/"
      },
      "require": {
        "junaidbhura/composer-wp-pro-plugins": "*"
      }
    }
  },
  {
    "type": "package",
    "package": {
      "name": "junaidbhura/wpml-cms-nav",
      "version": "<version_number>",
      "type": "wordpress-plugin",
      "dist": {
        "type": "zip",
        "url": "https://wpml.org/"
      },
      "require": {
        "junaidbhura/composer-wp-pro-plugins": "*"
      }
    }
  },
  {
    "type": "package",
    "package": {
      "name": "junaidbhura/wpml-translation-management",
      "version": "<version_number>",
      "type": "wordpress-plugin",
      "dist": {
        "type": "zip",
        "url": "https://wpml.org/"
      },
      "require": {
        "junaidbhura/composer-wp-pro-plugins": "*"
      }
    }

>   },
>   
> ],

Reviewer: Fix JSON syntax.

```suggestion
  }
],
```

For example:

`junaidbhura/wpae-acf-add-on`

> ### WPML Add-Ons
> 
> You can use any WPML add-on by simply adding it's slug like so:
> 
> `junaidbhura/<plugin-slug>`
> 
> For example:
> 
> `junaidbhura/wpml-sitepress-multilingual-cms`
> 
> `junaidbhura/wpml-string-translation`
> 
> `junaidbhura/wpml-media-translation`
> 
> `junaidbhura/wpml-woocommerce-multilingual`
> 
> `junaidbhura/wpml-gravityforms-multilingual`
> 
> `junaidbhura/wpml-contact-form-7-multilingual`
> 
> `junaidbhura/wpml-ninja-forms`
> 
> `junaidbhura/wpml-wpforms`
> 
> `junaidbhura/wpml-buddypress-multilingual`
> 
> `junaidbhura/wpml-acfml`
> 
> `junaidbhura/wpml-all-import`
> 
> `junaidbhura/wpml-mailchimp-for-wp`
> 
> `junaidbhura/wpml-wp-seo-multilingual`
> 
> `junaidbhura/wpml-types`
> 
> `junaidbhura/wpml-sticky-links`
> 
> `junaidbhura/wpml-cms-nav`
> 
> `junaidbhura/wpml-translation-management`

Reviewer: This section is derived from from WP All Import/Export Pro and both should be rephrased to highlight the fact that only specific add-ons of each plugin are supported. The section should be phrase as so:

```suggestion
### WPML Add-Ons

The following WPML plugins and add-ons are supported:

* `junaidbhura/wpml-acfml`
* `junaidbhura/wpml-all-import`
* `junaidbhura/wpml-buddypress-multilingual`
* `junaidbhura/wpml-cms-nav`
* `junaidbhura/wpml-contact-form-7-multilingual`
* `junaidbhura/wpml-gravityforms-multilingual`
* `junaidbhura/wpml-mailchimp-for-wp`
* `junaidbhura/wpml-media-translation`
* `junaidbhura/wpml-ninja-forms`
* `junaidbhura/wpml-sitepress-multilingual-cms`
* `junaidbhura/wpml-sticky-links`
* `junaidbhura/wpml-string-translation`
* `junaidbhura/wpml-translation-management`
* `junaidbhura/wpml-types`
* `junaidbhura/wpml-woocommerce-multilingual`
* `junaidbhura/wpml-wp-seo-multilingual`
* `junaidbhura/wpml-wpforms`
```
